sample-apps/javascript-sample-app/README.md

```diff
@@ -0,0 +1,18 @@
+## Javascript Opentelemetry Sample App
+
```

Comment:
Is this a manual or auto instrumentation application? can you include details in the readme?

Reply:
Fixed